### ISSUE TITLE
Json data question

### ISSUE THREAD

**Issue by ShaineRosewel** [Devographics/Monorepo#333]
Not sure if this is something expected. I am looking at the json data of the Oct 23 zip file sent by Sacha.

This response is from ID: "2U7I-Z8CHPSHrkZUf11tI"
<img width="398" alt="Screenshot 2023-10-27 at 10 05 39 PM" src="https://github.com/Devographics/Monorepo/assets/62248526/f2e739ec-698d-4b8f-86ec-c4920048d91c">

then I looked at another ID: "joLamwEOvW74Z0k8FFmt5"
<img width="633" alt="Screenshot 2023-10-27 at 10 10 34 PM" src="https://github.com/Devographics/Monorepo/assets/62248526/bab4ff5a-75b9-4399-a2f0-b3c75424a4d8">

I am wondering why there isn't a line, _"experience": "never_used "_ on the first image even if follow up is not empty... Is this expected? @SachaG @eric-burel

**Comment by eric-burel** [comment]
Might something slightly related to https://github.com/Devographics/Monorepo/issues/290? If you unclicked the element the followup would stay, I fixed a bug during the survey about that and the one in #290 specifically is still there (we clarified the expected behaviour but I didn't fix it yet)

I think you can ignore those results

**Comment by SachaG** [comment]
Yes in theory you always have to pick a "main" experience answer before you can pick a follow-up, but because of UI weirdness it might not always be the case (that "rule" is not enforced server-side so anomalies like that can get through).
